[2021-10-10 17:07:31,420] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_complex.create_entry_group_result2 2021-10-10T14:04:43.758814+00:00 [queued]>
[2021-10-10 17:07:31,424] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_complex.create_entry_group_result2 2021-10-10T14:04:43.758814+00:00 [queued]>
[2021-10-10 17:07:31,424] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-10 17:07:31,424] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-10 17:07:31,425] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-10 17:07:31,431] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): create_entry_group_result2> on 2021-10-10T14:04:43.758814+00:00
[2021-10-10 17:07:31,432] {standard_task_runner.py:52} INFO - Started process 16101 to run task
[2021-10-10 17:07:31,435] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_complex', 'create_entry_group_result2', '2021-10-10T14:04:43.758814+00:00', '--job-id', '97', '--pool', 'default_pool', '--raw', '--subdir', '/usr/local/lib/python3.8/dist-packages/airflow/example_dags/example_complex.py', '--cfg-path', '/tmp/tmp856j93g9', '--error-file', '/tmp/tmp7pm1r2js']
[2021-10-10 17:07:31,436] {standard_task_runner.py:77} INFO - Job 97: Subtask create_entry_group_result2
[2021-10-10 17:07:31,488] {logging_mixin.py:109} INFO - Running <TaskInstance: example_complex.create_entry_group_result2 2021-10-10T14:04:43.758814+00:00 [running]> on host abd
[2021-10-10 17:07:31,509] {taskinstance.py:1252} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_complex
AIRFLOW_CTX_TASK_ID=create_entry_group_result2
AIRFLOW_CTX_EXECUTION_DATE=2021-10-10T14:04:43.758814+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-10T14:04:43.758814+00:00
[2021-10-10 17:07:31,509] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-10-10 17:07:31,509] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo create_entry_group_result2']
[2021-10-10 17:07:31,513] {subprocess.py:74} INFO - Output:
[2021-10-10 17:07:31,514] {subprocess.py:78} INFO - create_entry_group_result2
[2021-10-10 17:07:31,514] {subprocess.py:82} INFO - Command exited with return code 0
[2021-10-10 17:07:31,527] {taskinstance.py:1212} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=create_entry_group_result2, execution_date=20211010T140443, start_date=20211010T140731, end_date=20211010T140731
[2021-10-10 17:07:31,566] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-10 17:07:31,608] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
